merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/UpdateActivityDirectiveValidationsAction.java

```diff
@@ -27,17 +22,30 @@ public UpdateActivityDirectiveValidationsAction(final Connection connection) thr
     this.statement = connection.prepareStatement(sql);
   }
 
-  public void apply(final long directiveId, final long planId, final Timestamp argumentsModifiedTime, final List<ValidationNotice> notices)
+  public void apply(final ActivityDirectiveId directiveId, final PlanId planId, BulkArgumentValidationResponse results)
```

Comment:
I debated passing a list in, but figured it might be useful to keep around a version that acted on a single validation. Thinking about it more, I suppose the bulk version can just be passed a list of size 1 if we did want this functionality. Change has been made.